docs/api/rest/dicomweb-stow.md

```diff
@@ -0,0 +1,85 @@
+<!--
+SPDX-FileCopyrightText: © 2022 MONAI Consortium
+SPDX-License-Identifier: Apache License 2.0
+-->
+
+# DICOMWeb STOW-RS APIs
+
+The `dicomweb/` endpoint implements the specifications defined in section [6.6 STOW-RS Request/Response](https://dicom.nema.org/dicom/2013/output/chtml/part18/sect_6.6.html#sect_6.6.1.3.2.1.1) defined by the DICOM committee to provide the [DICOMWeb STOW-RS](https://www.dicomstandard.org/using/dicomweb/store-stow-rs) interface to enable the triggering of new workflows. 
+
+
+The *STOW-RS* service provides the following two endpoints:
```

Comment:
Nit: maybe it is better to refer to resource type (one type) as specified in  https://dicom.nema.org/dicom/2013/output/chtml/part18/sect_6.6.html#sect_6.6.1.1, as it is a RESTful.

Reply:
Changed it to `...following two endpoints/resources:`